### ISSUE TITLE
Using jquery-ujs, ajaxSetup, and php in the javascript

### ISSUE THREAD

**Issue by rbruhn** [barryvdh/laravel-translation-manager#120]
Just a couple suggestions....

1) If you are going to use jquery.ujs it's probably better to include via cdn or bower.
2) You don't need the ajaxSetup if you simply use meta tags and let jquery-ujs handle it:

```
<meta name="csrf-param" content="_token">
<meta name="csrf-token" content="{{ csrf_token() }}">
```

3) In regard to php in the javascript, you can use data-route in the group-select option tag, then pick that up when the option is selected.

```
<option data-route="{{ $key === '' ? route('admin.translations.index') : route('admin.translations.view') }}" value="{{  $key }}"{{  $key == $group ? ' selected':'' }}>{{  $value }}</option>

    $('.group-select').on('change', function () {
        var group = $(this).val();
        var url = $(this).find(':selected').data('route')
        if (group) {
            window.location.href = url + '/' + $(this).val();
        } else {
            window.location.href = url;
        }
    });
```

**Comment by barryvdh** [comment]
Why is it better to use a cdn? With local js I'm sure it keeps working. 

I know about the meta tags, I even wrote a blog about it :p But I think I still needed them for something. Perhaps the x-editable. Not really sure though.

Feel free to submit a pr for the php/js

**Comment by rbruhn** [comment]
Didn't mean my post to come across as criticizing. Sorry if it did. Your package actually helped me a lot in handling those language files.

I simply suggested CDN because you use it for JQuery, X-editable, and Boostrap.

As for a pull request, I could do one but I changed a number of things so the index page would use blade templating as I showed above. I did remove the ajaxSetUp in your code after adding the meta tags and everything worked fine.

It seems there are a lot of pull requests in your queue, and I'd be more than happy to contribute if you decide what direction you'd like to take this package. It seems a lot of people are interested in it.

I use a lot of other packages you've created and you always do great work :)
